mission_control/navigator_missions/nav_missions/coral_survey.py

```diff
@@ -19,8 +19,9 @@
 def main(navigator):
     result = navigator.fetch_result()
 
-    middle_point = np.array([-10, -70, 0]) #yield navigator.database_query("coral_survey")
-
+    #middle_point = np.array([-10, -70, 0]) 
+    resp = yield navigator.database_query("start_gate")
```

Comment:
You should check ```resp.found  == True``` here to be safe. If the database doesn't have the start gate in it, this will throw an out of index.